package.json

```diff
@@ -3,11 +3,15 @@
   "description": "A monorepo containing all things frontend on Alephium",
   "author": "Alephium dev <dev@alephium.org>",
   "packageManager": "bun@1.0.9",
+  "private": true,
   "workspaces": [
     "apps/*",
     "packages/*"
   ],
   "scripts": {
+    "lint": "turbo lint",
+    "test": "turbo test",
+    "typecheck": "turbo typecheck",
```

Comment:
These commands are mainly needed so that the CI can run them. But, of course, we can also run them locally, even though we will have turbo available locally anyway.